elements/pfe-primary-detail/src/pfe-primary-detail.js

```diff
@@ -300,16 +339,134 @@ class PfePrimaryDetail extends PFElement {
         details: nextDetails
       }
     });
+  } // end _handleHideShow()
 
-    // Set focus to pane
-    const firstFocusableElement = nextDetails.querySelector(
-      'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
-    );
-    if (firstFocusableElement) {
-      firstFocusableElement.focus();
+  /**
+   * A11y features
+   */
+  _isToggle(element) {
+    return element.classList.contains("a11y-toggle");
```

Comment:
Could do `return element.getAttribute('slot') === "details-nav"`?

Reply:
@wesruv will that return the button tag itself or only the details nav wrapper?

Reply:
@wesruv never mind, I was confused about which part we were discussing. I understand this now.